terminal.c

```diff
@@ -6502,6 +6508,14 @@ void term_mouse(Terminal *term, Mouse_Button braw, Mouse_Button bcooked,
                       !(term->mouse_override && shift));
     int default_seltype;
 
+    //   Don't do anything if mouse movement events weren't requested;
+    //   Note: return early to avoid doing all of this code on every mouse move
+    //   event only to throw it away. ?
```

Comment:
Just drop the question mark here - that comment is fine.

And for all "checked" comments please adjust to `/* comment */` style as it seems to be used in the existing file.